src/types/profile-derived.js

```diff
@@ -393,10 +393,10 @@ export type TracedTiming = {|
  * https://searchfox.org/mozilla-central/rev/3811b11b5773c1dccfe8228bfc7143b10a9a2a99/tools/profiler/core/platform.cpp#3000-3186
  */
 export type EventDelayInfo = {|
-  +eventDelays: Milliseconds[],
-  +minDelay: Milliseconds,
-  +maxDelay: Milliseconds,
-  +delayRange: Milliseconds,
+  +eventDelays: Float32Array[],
+  +minDelay: Float32Array,
+  +maxDelay: Float32Array,
+  +delayRange: Float32Array,
```

Comment:
These type changes doesn't look correct. Only `eventDelays` field should be changed to `Float32Array`. And Float32Array is an array by itself already, we don't need `[]` for that.